src/init/init.c

```diff
@@ -20,7 +21,9 @@
 #include <sof/schedule/task.h>
 #include <sof/sof.h>
 #include <sof/trace/trace.h>
+#include <ipc/topology.h>
 #include <ipc/trace.h>
+#include <xtos-structs.h>
```

Comment:
This is an architecture only header so cant be included here, can only be included under arch/xtensa/